stdlib/hmac.pyi

```diff
@@ -21,6 +21,7 @@ else:
     def new(key: bytes, msg: ReadableBuffer | None = ..., digestmod: _DigestMod | None = ...) -> HMAC: ...
 
 class HMAC:
+    blocksize: ClassVar[int]
     digest_size: int
     block_size: int
```

Comment:
I don't think we should add `blocksize`, because instances already have `block_size`, which is the documented way to do this.